app/controllers/Application.scala

```diff
@@ -171,80 +172,78 @@ object Application extends Controller {
-      }.collect {
-        case Some(x) => x
-      }.map { case (k, kernel) =>
-        val path = kernel.notebookPath.getOrElse(s"KERNEL '$k' SHOULD HAVE A PATH ACTUALLY!")
-        Json.obj(
-          "notebook" →  Json.obj(
-                          "path" → path
-                        ),
-          "id" →  k
-        )
-      }.toSeq)
+      KernelManager.get(k).map(l => (k, l))
+    }.collect {
+      case Some(x) => x
+    }.map { case (k, kernel) =>
+      val path: String = kernel.notebookPath.getOrElse(s"KERNEL '$k' SHOULD HAVE A PATH ACTUALLY!")
+      Json.obj(
+        "notebook" → Json.obj("path" → path),
+        "id" → k
+      )
+    }.toSeq)
     )
   }
 
 
   def profiles() = Action.async {
     implicit val ec = kernelSystem.dispatcher
-    (clustersActor ? NotebookClusters.Profiles).map { case all:List[JsObject] =>
+    (clustersActor ? NotebookClusters.Profiles).map { case all: List[JsObject] =>
       Ok(JsArray(all))
     }
   }
 
   def clusters() = Action.async {
     implicit val ec = kernelSystem.dispatcher
-    (clustersActor ? NotebookClusters.All).map { case all:List[JsObject] =>
+    (clustersActor ? NotebookClusters.All).map { case all: List[JsObject] =>
       Ok(JsArray(all))
     }
   }
 
   /**
-   *  {
-   *    "name": "Med At Scale",
-   *    "profile": "Local",
-   *    "template": {
-   *      "customLocalRepo": "/tmp/spark-notebook/repo",
-   *      "customRepos": [],
-   *      "customDeps": [
-   *        "org.bdgenomics.adam % adam-apis % 0.15.0",
-   *        "- org.apache.hadoop % hadoop-client %   _",
-   *        "- org.apache.spark  % spark-core    %   _",
-   *        "- org.scala-lang    %     _         %   _",
-   *        "- org.scoverage     %     _         %   _",
-   *        "+ org.apache.spark  %  spark-mllib_2.10  % 1.2.0"
-   *      ],
-   *      "customImports": [
-   *        "import org.apache.hadoop.fs.{FileSystem, Path}",
-   *        "import org.bdgenomics.adam.converters.{ VCFLine, VCFLineConverter, VCFLineParser }",
-   *        "import org.bdgenomics.formats.avro.{Genotype, FlatGenotype}",
-   *        "import org.bdgenomics.adam.models.VariantContext",
-   *        "import org.bdgenomics.adam.rdd.ADAMContext._",
-   *        "import org.bdgenomics.adam.rdd.variation.VariationContext._",
-   *        "import org.bdgenomics.adam.rdd.ADAMContext",
-   *        "import org.apache.spark.rdd.RDD"
-   *      ],
-   *      "customSparkConf": {
-   *        "spark.app.name": "Local Adam Analysis",
-   *        "spark.master": "local[8]",
-   *        "spark.executor.memory": "1G",
-   *        "spark.serializer": "org.apache.spark.serializer.KryoSerializer",
-   *        "spark.kryo.registrator": "org.bdgenomics.adam.serialization.ADAMKryoRegistrator",
-   *        "spark.kryoserializer.buffer.mb": "4",
-   *        "spark.kryo.referenceTracking": "true",
-   *        "spark.executor.memory": "2g"
-   *      }
-   *    }
-   *  }
+   * {
```

Comment:
Wouldn't be better keeping the pretty printed json here :-D? Although, this example could also be simply removed since the documentation explains that